transform/snowflake-dbt/models/mattermost/downloads.sql

```diff
@@ -4,39 +4,121 @@
+                 ELSE NULL END                                                                     AS version
+           , nullif(regexp_replace(
+                            regexp_replace(regexp_replace(split_part(log_entries.creferrer, '/', 3), '^[w]{3}[.]', '')
+                                , '.com[0-9_a-z_A-Z:.]{1,}$', '.com'), ':{1}[0-9]{1,4}$', ''), '') AS server_source
+           , log_entries.edge
+           , log_entries.bytessent
+           , log_entries.method
+           , log_entries.host
+           , log_entries.status
+           , log_entries.creferrer
+           , log_entries.useragent
+           , log_entries.cs_uri_query
+           , log_entries.cookie
+           , log_entries.x_edge_result_type
+           , log_entries.x_edge_request_id
+           , log_entries.x_host_header
+           , log_entries.protocol
+           , log_entries.cs_bytes
+           , log_entries.time_taken
+           , log_entries.x_forwarded_for
+           , log_entries.ssl_protocol
+           , log_entries.ssl_cipher
+           , log_entries.x_edge_response_result_type
+           , log_entries.cs_protocol_version
+           , log_entries.file_status
+           , log_entries.file_encrypted_fields
+           , o.diff                                                                                AS bytessent_diff_from_uri_mean
+           , o.std                                                                                 AS bytessent_std_from_uri_mean
+           , o.avg                                                                                 AS bytessent_uri_avg                                                                              AS bytessent_uri_avg
+         FROM {{ source('releases', 'log_entries') }} log_entries
+              JOIN outliers o
+                   ON log_entries.logdate::DATE = o.logdate::DATE
+                       AND log_entries.cip = o.cip
+                       AND log_entries.uri = o.uri
+                       AND log_entries.bytessent = o.bytessent
+                       AND log_entries.logtime = o.logtime
+         WHERE (regexp_like(log_entries.uri, '^\/[1-9]{1,3}[.][0-9]{1,3}[.][0-9]{1,3}\/.*') OR
+                regexp_like(log_entries.uri, '^\/desktop\/[1-9]{1,3}[.][0-9]{1,3}[.][0-9]{1,3}\/.*'))
+           AND log_entries.bytessent > 1000000
+           AND log_entries.status LIKE '2%'
+           AND NOT o.one_std_from_mean
+           {% if is_incremental() %}
 
-    {% endif %}
-    GROUP BY logdate, cip, type
-)
+            -- this filter will only be applied on an incremental run
+            AND log_entries.logdate::date > (SELECT MAX(logdate) FROM {{ this }})
 
-SELECT * FROM downloads
\ No newline at end of file
+          {% endif %}
+         GROUP BY 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21
+                , 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34
+     )
+
+SELECT *
+FROM server_downloads
```

Comment:
Is this just **Server Downloads** or is it all?

Reply:
@rbradleyhaas Already updated this prior to seeing your comment. It's all.